### DISCUSSION TITLE
Can Not Run start_ofd.py

### DISCUSSION THREAD

**Discussion by ghost** [UltimaHoarder/UltimaScraper#1885] [General]
Hello, using windows 10 and trying to launch start_ofd.py I get the message

"Execute the script with Python 3.9"

Also I just tried with updating Python to 3.10 and the same thing happens.

Anyone else experiencing this issue?

**Comment by loko664** [comment]
yeah me too whats the fix for this?

**Comment by akd2001** [comment]
this script works in python3.9.
#889
